src/utils/config.ts

```diff
@@ -300,8 +300,15 @@ export const FILE_ITEM_PLUGIN_OPTIONS = { storageRootPath: FILE_STORAGE_ROOT_PAT
 
 export const ITEMS_ROUTE_PREFIX = '/items';
 export const APP_ITEMS_PREFIX = '/app-items';
+export const SHORT_LINKS_ROUTE_PREFIX = '/short-links';
 export const THUMBNAILS_ROUTE_PREFIX = '/thumbnails';
 
+// Graasp view (builder, player, library) consts
+export const BUILDER_ITEMS_PREFIX = ITEMS_ROUTE_PREFIX;
```

Comment:
I think we should decouple the frontend route from the backend route. So use a new constant here, it may have the same value, but the name should clearly state the purpose of the constant.